README.md

```diff
@@ -466,27 +420,136 @@ When using this functionality, every `_tmp` table should use this macro as descr
 ```
+**Output:**
+```sql
+{% macro get_admin_columns() %}
+
+{% set columns = [
+    {"name": "email", "datatype": dbt_utils.type_string()},
+    {"name": "id", "datatype": dbt_utils.type_string(), "alias": "admin_id"},
+    {"name": "job_title", "datatype": dbt_utils.type_string()},
+    {"name": "name", "datatype": dbt_utils.type_string()},
+    {"name": "_fivetran_deleted", "datatype": "boolean"},
+    {"name": "_fivetran_synced", "datatype": dbt_utils.type_timestamp()}
+] %}
 
+{{ return(columns) }}
+
+{% endmacro %}
+```
 **Args:**
-* `union_schema_variable` (optional): The name of the union schema variable. By default the macro will look for `union_schemas`.
-* `union_database_variable` (optional): The name of the union database variable. By default the macro will look for `union_databases`.
+* `table_name`    (required): Name of the schema which the table you are running the macro for resides in.
+* `schema_name`   (required): Name of the schema which the table you are running the macro for resides in.
+* `database_name` (optional): Name of the database which the table you are running the macro for resides in. If empty, the macro will default this value to `target.database`.
 
-### add_dbt_source_relation ([source](macros/add_dbt_source_relation.sql))
-This macro is intended to be used within the second CTE (typically named `fields`) of non-tmp staging models. 
-It simply passes through the `_dbt_source_relation` column produced by `union_data()` in the tmp staging model, so that `source_relation()` can work in the final CTE of the staging model.
+----
+### get_columns_for_macro ([source](macros/get_columns_for_macro.sql))
+This macro returns all column names and datatypes for a specified table within a database and is used as part of the [generate_columns_macro](macros/generate_columns_macro.sql).
 
 **Usage:**
 ```sql
-{{ fivetran_utils.add_dbt_source_relation() }}
+{{ fivetran_utils.get_columns_for_macro(table_name="team", schema_name="my_teams", database_name="my_database") }}
+```
+**Args:**
+* `table_name`    (required): Name of the table you are wanting to return column names and datatypes.
+* `schema_name`   (required): Name of the schema where the above mentioned table resides.
+* `database_name` (optional): Name of the database where the above mentioned schema and table reside. By default this will be your target.database.
+
+----
+### staging_models_automation ([source](macros/staging_models_automation.sql))
+This macro is intended to be used as a `run-operation` when generating Fivetran dbt source package staging models/macros. This macro will receive user input to create the necessary bash commands using ([generate_columns](generate_columns.sh)) and ([generate_models](generate_models.sh)) appended with `&&` so they may all be ran at once. The output of this macro within the CLI will then be copied and pasted as a command to generate the staging models/macros.
```

Comment:
Please add the below:
Additionally, you can rerun this macro as it will create or replace what is currently in the macro & model folders. 

Things to note:
- This macro will only work if you have already included your `src.yml` file. 
- Please double check your outputs as there may be timestamps & notes that are not relevant to the file.